bindings/pulp/bindings/server_info.py

```diff
@@ -51,12 +52,3 @@ def get_distributors(self):
         """
         path = self.base_path + 'distributors/'
         return self.server.GET(path)
-
-    def ping(self):
-        """
-        Retrieves basic status information from the server.
-
-        @return: Response
-        """
-        path = '/v2/services/status/'
-        return self.server.GET(path)
```

Comment:
Since the bindings could be considered part of our API, we shouldn't remove this until 3.0.0. However, we could insert a DeprecationWarning.